src/scss/App.scss

```diff
@@ -67,7 +87,7 @@ body {
       color: $white;
       font-family: spurs;
       font-size: 22px;
-      letter-spacing: 1px;
+      letter-spacing: 2px;
```

Comment:
Should be `letter-spacing 1px`. It seems some css changes from PR #42 got pushed into this one. I'd be cautious about mixing PR codes. We'll discuss offline.

Reply:
Cool. I'm starting to see that myself. I'll be coding along and think it would be good to fix this little bit of code since I'm here. But prolly not a good idea, better to deal with it in its own branch.